Docs/user_guide/about.md

```diff
@@ -18,10 +18,34 @@ Highlights:
 
 ## License
 
-The 3D Slicer software is distributed under a BSD-style open source license that is compatible with the Open Source Definition by [The Open Source Initiative](http://opensource.org/) and contains no restrictions on use of the software.
+The 3D Slicer software is distributed under a BSD-style open source license that is broadly compatible with the Open Source Definition by [The Open Source Initiative](http://opensource.org/) and contains no restrictions on legal uses of the software.
 
 To use Slicer, please read the [3D Slicer Software License Agreement](https://github.com/Slicer/Slicer/blob/master/License.txt) before downloading any binary releases of the Slicer.
 
+### Historical notes about the license
+
+The Slicer License was drafted in 2005 by lawyers working for Brigham and Women's Hospital (BWH), a teaching affiliate of Harvard Medical School, to be BSD-like but with a few extra provisions related to medical software.  It is specific to BWH so it's not directly reusable, but it could serve as a template for projects with similar goals.
+
+It was written in part because BWH was the prime contractor on an NIH-funded development consortium ([NA-MIC](na-mic.org)) and wanted all code contributions to be compatible with ultimate use in real-world medical products (that is, commercial FDA-approved medical devices, which are almost universally closed source even if they build on open software).  Compliance with the Slicer License was required for subcontractors, a group that included GE Research, Kitware and several universities (MIT, UNC...) who all reviewed and accepted this license.
+
+The license has been in continuous use since 2005 for the 3D Slicer software package (slicer.org) that as of 2021 has been downloaded more than a million times and has been referenced in about 12,000 academic publications (https://www.slicer.org/wiki/Main_Page/SlicerCommunity). Some of the code is also now being used in several medical products for which this license has been reviewed and accepted by the companies involved.
+
+### License terms and reasons
+
+Here are some of the key points that BWH included to make the license suit the case of a large hospital distributing open source medical software.
+
+* Contributors affirm that they have de-identified any patient data they contribute to avoid issues with HIPAA or related regulations.
```

Comment:
I like the whole set of changes in this pull request very much. It is very clear and accurate description of the license.

I would just recommend to separate in this section more clearly terms that affect users and terms that are only for contributors. We should start with users, as most people are just users.

Reply:
We were waiting for the OSI team to summarize their discussions to their wiki, but that appears to take longer than expected.  I would say let's finalize and merge this.  We can always update it later as needed.

Reply:
Agreed.